### ISSUE TITLE
Is there a flag to get validation(test) accuracy in caffe2?

### ISSUE THREAD

**Issue by jeageun** [facebookresearch/dlrm#37]
Hi, I want to ask something about the code.

I found that on PyTorch version, there is flag '--test-freq' to get validation accuracy, but I can't figure it out in the caffe2 version. Is there the same flag on caffe2 for this purpose? 

Thanks,
Jeageun

**Comment by mnaumovfb** [comment]
The --test-freq flag is not supported for caffe2 version at the moment.

We could potentially add it in the future, but it would require creating additional caffe2 net without the backward operators and using it for testing.

**Comment by jeageun** [comment]
Then, is there any trick, or alternative way to get validation results? I want to use Caffe2 version because it's faster than Pytorch.

**Comment by mnaumovfb** [comment]
Unfortunately, not at the moment. I can take it as a request for the future.

If you are only interested in accuracy/loss numbers then the pytorch and caffe2 versions should be essentially the same. However, you would need to run pytorch version to get it.

**Comment by mnaumovfb** [comment]
We have added support for --test-freq flag in caffe2 version. 

Please take a look and let me know if it works for you.
